<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
    <widget class="QWidget" name="myRecordsPage">
     <widget class="QTableWidget" name="myRecordTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>781</width>
        <height>700</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Cover</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Record</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Artist</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rating</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tags</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="myRecordsPageLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>My Collection</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toSearchRecordsButton">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>15</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Find New Records</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeFromMyRecordButton">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>670</y>
        <width>191</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Remove from
My Collection</string>
      </property>
     </widget>
     <widget class="QSlider" name="myRecordRatingSlider">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>620</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
     <widget class="QLabel" name="myRecordRatingNumLabel1">
      <property name="geometry">
       <rect>
        <x>802</x>
        <y>640</y>
        <width>120</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0   1   2   3   4</string>
      </property>
     </widget>
     <widget class="QLabel" name="myRecordSortLabel">
      <property name="geometry">
       <rect>
        <x>222</x>
        <y>13</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sort by:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="myRecordSearchBar">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>15</y>
        <width>321</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Search My Collection</string>
      </property>
     </widget>
     <widget class="QComboBox" name="myRecordSortBox">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>15</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Oldest to Newest</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Newest to Oldest</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Title Ascending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Title Descending </string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Artist Ascending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Artist Descending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rating Ascending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rating Descending</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="myRecordRatingLabel">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>600</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Record Rating:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="myRecordEditTagsList">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>420</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
     <widget class="QLabel" name="myRecordEditTagsLabel">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>397</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Toggle Tags:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="myRecordSortTagsList">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>110</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="myRecordSortTagsLabel">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>90</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sort by Tags:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="myRecordManageTagButton">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>388</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manage Tags</string>
      </property>
     </widget>
     <widget class="QLabel" name="myRecordRatingNumLabel2">
      <property name="geometry">
       <rect>
        <x>892</x>
        <y>640</y>
        <width>120</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5   6   7   8   9  10</string>
      </property>
     </widget>
     <widget class="QLabel" name="myRecordRecordCountLabel">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>758</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Showing 0/0 Records</string>
      </property>
     </widget>
     <widget class="QPushButton" name="myRecordPickForMe">
      <property name="geometry">
       <rect>
        <x>635</x>
        <y>755</y>
        <width>151</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pick a Record For Me</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="searchRecordsPage">
     <widget class="QTableWidget" name="searchRecordTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>781</width>
        <height>700</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Cover</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Record</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Artist</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="searchPageLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Search New Records</string>
      </property>
     </widget>
     <widget class="QPushButton" name="toMyRecordsButton">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>15</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Back to My
Collection</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchRecordSearchBar">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>15</y>
        <width>321</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Search For New Records</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addToMyRecordButton">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>670</y>
        <width>191</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add to My
Collection</string>
      </property>
     </widget>
     <widget class="QLabel" name="searchRecordsInfoLabel">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>753</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="searchRecordRatingLabel">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>600</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Record Rating:</string>
      </property>
     </widget>
     <widget class="QSlider" name="searchRecordRatingSlider">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>620</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
     <widget class="QLabel" name="searchRecordRatingNumLabel1">
      <property name="geometry">
       <rect>
        <x>892</x>
        <y>640</y>
        <width>120</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>5   6   7   8   9  10</string>
      </property>
     </widget>
     <widget class="QLabel" name="searchRecordRatingNumLabel2">
      <property name="geometry">
       <rect>
        <x>802</x>
        <y>640</y>
        <width>120</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0   1   2   3   4</string>
      </property>
     </widget>
     <widget class="QListWidget" name="searchRecordSuggestedTagsList">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>420</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="searchRecordSuggestedTagsLabel">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>397</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Suggested Tags:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionContact"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionToggle_Theme"/>
    <addaction name="actionImport_Records"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionToggle_Theme">
   <property name="text">
    <string>Toggle Theme</string>
   </property>
  </action>
  <action name="actionyes">
   <property name="text">
    <string>yes</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImport_Records">
   <property name="text">
    <string>Import Records</string>
   </property>
  </action>
  <action name="actionContact">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
